src/main/deltachat.js

```diff
@@ -307,7 +312,9 @@ class DeltaChatController {
     for (let i = 0; i < count; i++) {
       const chatId = list.getChatId(i)
       const chat = chatIdToJson(chatId, this._dc)
+      const freshMessageCounter = this._dc.getFreshMessageCount(chatId)
       chat.summary = list.getSummary(i).toJson()
+      chat.freshMessageCounter = freshMessageCounter
```

Comment:
instead, let's put this in the chatIdToJson function. The only reason we have `chat.summary` here is because we need `list` in order to `getSummary` on the chat.

```
chat.freshMessageCounter = this._dc.getFreshMessageCount(chatId)
```